src/settings/SettingsModal.tsx

```diff
@@ -121,6 +122,16 @@ export const SettingsModal: FC<Props> = ({
             title={t("settings.devices.speaker")}
             numberedLabel={(n) => t("settings.devices.speaker_numbered", { n })}
           />
+          <FieldRow>
+            <InputField
```

Comment:
This is more of a product nit, but I'm surprised we don't just expose a pan setting like most volume controls do.